pkg/kubectl/cmd/drain.go

```diff
@@ -98,6 +108,8 @@ func NewCmdCordon(f cmdutil.Factory, out io.Writer) *cobra.Command {
 			cmdutil.CheckErr(options.RunCordonOrUncordon(true))
 		},
 	}
+	cmd.Flags().StringVar(&options.Message, "set-message", "", setMessageDesc)
```

Comment:
Naming nit: I prefer `--message` or `--reason` as a flag name.  We generally only use verb-type flag names when they toggle behavior , e.g. `--match-server-version=false`.